docs/ref/settings.txt

```diff
@@ -371,6 +371,33 @@ Whether to use a secure cookie for the CSRF cookie. If this is set to ``True``,
 the cookie will be marked as "secure," which means browsers may ensure that the
 cookie is only sent with an HTTPS connection.
 
+.. setting:: CSRF_USE_SESSIONS
+
+CSRF_USE_SESSIONS
+-----------------
+
+.. versionadded:: 1.11
+
+Default: ``False``
+
+Whether to store the CSRF token inside a the user's session instead of
+a cookie. This requires that you are using Django's session module, see
```

Comment:
This requires the use of sessions, see :mod:`django.contrib.sessions` for more information

(the point is that while some session implementation is required, it need not be Django's; but we don't really need to get into this nuance, just avoid implying that the session must be Django's)